src/controller/rpc.cpp

```diff
@@ -91,6 +91,7 @@ class RPCRequest {
 
 RPC::RPC(int ipc_fd, const CommandApi::Config& config) : ipc_fd(ipc_fd) {
     this->api = std::make_unique<CommandApi>(config, *this);
+    this->rpc_timeout = std::chrono::milliseconds(config.controller_rpc_timeout_ms);
 }
```

Comment:
using the member initialization list (`ipc_fd(ipc_fd), rpc_timeout(std::chrono::milliseconds(config.controller_rpc_timeout_ms))` should be best practice